src/main/java/com/laserfiche/repository/api/clients/impl/model/Entry.java

```diff
@@ -10,6 +12,8 @@
 import java.util.Objects;
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2022-08-19T17:07:43.799-04:00[America/New_York]")
+@JsonTypeInfo(use = JsonTypeInfo.Id.DEDUCTION)
+@JsonSubTypes({@JsonSubTypes.Type(Folder.class), @JsonSubTypes.Type(Shortcut.class)})
```

Comment:
I know that the tests pass, but just to improve the code, isn't it better to use fully qualified class names. I am not sure how Folder.class is resolved.

Reply:
Hi Samad, that's a good catch. But I think I know how it's resolved: in this case, it doesn't need an import statement or fully qualified class name because the Entry class and its subtypes are in the same package.